src/testsuite/vec_int128_dummy.c

```diff
@@ -441,6 +623,110 @@ test_mul4uq (vui128_t *__restrict__ mulu, vui128_t m1h, vui128_t m1l, vui128_t m
   mulu[3] = mphh;
 }
 
+/* alternative algorithms tested and not selected due to code size
+   and cycle latency.  */
```

Comment:
is it worth mentioning these in any more detail, so others won't bother trying the same approaches?